.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,34 @@
+Name: Clean up abandoned files
+
+on:
+  schedule:
+    - cron: '30 0 * * 1'
+  push:
+    branches:
+    - master
```

Comment:
If the action runs when a push is made to `master`, then I don't see any reason to schedule it as a cron job.

Reply:
right - I think keeping the cron-job only is probably best to make this more manageable